src/sqlancer/postgres/PostgresProvider.java

```diff
@@ -325,17 +325,24 @@ protected void prepareTables(PostgresGlobalState globalState) throws Exception {
     private String getCreateDatabaseCommand(PostgresGlobalState state) {
         StringBuilder sb = new StringBuilder();
         sb.append("CREATE DATABASE " + databaseName + " ");
-        if (Randomly.getBoolean() && ((PostgresOptions) state.getDbmsSpecificOptions()).testCollations) {
```

Comment:
Does the omission of `Randomly.getBoolean()` change the behavior of this code? The original idea of this was that only with some chance will we create collation-related things, even if the collation-option is enabled (i.e., the option allows creating collation settings, but it doesn't mandate them).

Reply:
Correct - For the default setting (`--test-collations=true`) that behavior should not change.

The way I ensured current default behavior is that if `--test-collation` flag is enabled, there would be a `Randomly.getBoolean()` to check if we're setting the collation options... and within that `if`, there's a second `Randomly.getBoolean()` to check if we're also doing `WITH ENCODING`. IIUC, this is how things were before this patch.